addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
Makes sense to me. Also I noticed that the VSCode snippets by @johnpapa declare `ngOnInit` as well.

On the other hand, I haven't used this hook that often in the components that I've developed and if I was using the CLI in most cases I'd have to remove the `ngOnInit` declaration.